roles/remove-node/pre-remove/tasks/main.yml

```diff
@@ -3,25 +3,13 @@
   set_fact:
     nodes_to_drain: []
 
-- name: remove-node | Identify nodes to drain, ignore non-cluster nodes
-  shell: |
-    set -o pipefail
-    {{ bin_dir }}/kubectl get nodes -o json \
-                  | jq .items[].metadata.name \
-                  | jq "select(. | test(\"^{{ hostvars[item]['kube_override_hostname']|default(item) }}$\"))"
-  loop: "{{ node.split(',') | default(groups['kube-node']) }}"
+- name: remove-node | List nodes
+  command: kubectl get nodes -o go-template={% raw %}'{{ range .items }}{{ .metadata.name }}{{ "\n" }}{{ end }}'{% endraw %}
```

Comment:
When the above command locally, the following error is output
```
$ kubectl get nodes -o go-template={% raw %}'{{ range .items }}{{ .metadata.name }}{{ "\n" }}{{ end }}'{% endraw %}
{%Error from server (NotFound): nodes "raw" not found
invalid resource name "%}{{ range .items }}{{ .metadata.name }}{{ \"\\n\" }}{{ end }}{%": [may not contain '%']
Error from server (NotFound): nodes "endraw" not found
invalid resource name "%}": [may not contain '%']
```
After changing here like
```
kubectl get nodes -o go-template='{% raw %}{{ range .items }}{{ .metadata.name }}{{ "\n" }}{{ end }}{% endraw %}'
```
by changing ' positions, the command outputs the list of nodes.

Reply:
Ansible (Jinja) will remove the `{% raw %}` / `{% endraw %}`, I can use `!unsafe` if it's more clear

Reply:
Or I can put the `{% raw %}` / `{% endraw %}` around the whole line

Reply:
Hum I don't think you have to change anything, Ansible wil ltake care of %raw%/%endraw%, 
Other commands with raw/endraw are also not 'run-able' as is and require ansible processing.

Reply:
@oomichi your call, I don't have strong opinions on this one

Reply:
Thanks for your explanation, I just wanted to confirm that works fine as expected.
It is fine to keep this as it if working.

/lgtm